modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js

```diff
@@ -113,58 +113,31 @@ const _isObjectStrict = (value) =>
 /**
  * Returns URL with proper search params.
  */
-const _buildParams = ({
-	item = null,
-	itemValueKey,
-	portletNamespace,
-	redirectURL,
-	selection,
-	urlParamName,
-}) => {
-	return addParams(
-		item
-			? `${portletNamespace}${urlParamName}=${
-					itemValueKey ? item[itemValueKey] : JSON.stringify(item)
-			  }`
-			: `${portletNamespace}${urlParamName}=${selection[itemValueKey]}`,
-		redirectURL
-	);
+const _getRedirectURLWithParams = ({data, portletNamespace, selection}) => {
+	const {itemValueKey, redirectURL, urlParamName} = data;
+
+	return [selection].flat().reduce((acc, item) => {
```

Comment:
@claraizquierdo this way we do not have to check what is coming, we use always an array, and in case it is already an array, we just flat the first level. Seems to work fine, but I need to test more